.github/workflows/phpunit.yml

```
@@ -20,6 +20,7 @@ jobs:
           - "7.3"
           - "7.4"
           - "8.0"
+          - "8.1"
```

I guess alpine `8.1` tag will not be available without a `-rc` suffix before the official release of PHP 8.1 stable version. So if you want to run the build now, you have to add this suffix.
```suggestion
          - "8.1-rc"
```

---

Maybe "8.1.0RC3" is even more accurate. (not sure about it)

---

Ping @Ocramius